translations/translations.rc

```diff
@@ -1,41 +0,0 @@
-winsparkle_af      MOFILE  "af.mo"
```

Comment:
This does not "Fix compile errors with translations", this completely breaks the translations.

Reply:
I didn't know this would make it into the pull request. At any rate, visual studio 2019 complains about not finding the translation files when building vanilla win sparkle, even though the files exists. Seems like something wrong with the original solution/vc project.

Reply:
@vslavik Can you suggest a fix for this error? I could submit an issue on your repo but we are already here, so...

Reply:
> @vslavik Can you suggest a fix for this error?

That's the thing… *what* error? `master` builds [just fine](https://ci.appveyor.com/project/vslavik/winsparkle/builds/35923994).

Reply:
Visual studio complains about not finding the translation files listed in the resource file even though they exist. That happened in the zip source download as well as a clone from master.

Reply:
Again, see the link I provided for evidence to the contrary. If you believe there is genuine issue (on master, not on anything older), please report it, including all relevant details (including saying what you *did*), but I rather suspect from the overall tone of ad41ca4 that you opened the wrong (and converted) project file instead of using one directly usable by VS2019. There's only one project file on the `master` now, so that's a mistake no longer possible to make.

Reply:
Got it. I'll have a look at it again as I work on a pull request and report an issue on your repo if I can reproduce it.